fala/templates/adviser/accessibility_statement.html

```diff
@@ -0,0 +1,117 @@
+{% extends 'base.html' %}
+
+{% block content %}
+  <div class="govuk-grid-column-two-thirds">
+    <div id="google_translate_element" class="govuk-!-margin-bottom-6"></div>
+
+    <h1 class="govuk-heading-xl">Accessibility statement for Find a legal advisor</h1>
+
+    <p class="govuk-body">
+      This accessibility statement applies to the ‘Find a legal advisor’ service.
+    </p>
+
+    <p class="govuk-body">
+      This website is run by the <a class="govuk-link" href="https://www.gov.uk/government/organisations/legal-aid-agency" target="_blank" rel="noopener external noreferrer">Legal Aid Agency</a> (<abbr>LAA</abbr>), part of the Ministry of Justice (<abbr>MoJ</abbr>).
+      We want as many people as possible to be able to use this website. For example, that means you should be able to:
+    </p>
+
+    <ul class="govuk-list govuk-list--bullet">
+      <li>change colours, contrast levels and fonts using browser or device settings</li>
+      <li>zoom in up to 400% without the text spilling off the screen</li>
+      <li>navigate most of the website using a keyboard or speech recognition software</li>
+      <li>listen to most of the website using a screen reader (including the most recent versions of JAWS, NVDA and VoiceOver)</li>
+    </ul>
+
+    <p class="govuk-body">
+      We’ve also made the website text as simple as possible to understand.
+    </p>
+
+    <p class="govuk-body">
+      <a class="govuk-link" href="https://mcmw.abilitynet.org.uk/" target="_blank" rel="noopener external noreferrer">AbilityNet</a> has
```

Comment:
Checked with content designer: the links within the accessibility statement and privacy policy should not open in a new tab.